region/maridia/outer/Fish Tank.json

```diff
@@ -693,6 +672,22 @@
       "notable": false,
       "requires": []
     },
+    {
+      "link": [4, 5],
+      "name": "Shinespark",
+      "notable": false,
+      "entranceCondition": {
```

Comment:
Is it a probalem that this is a spawnAt?

Reply:
Talked to @blkerby about this. Looks like this should work fine .